### DISCUSSION TITLE
An engineering machine

### DISCUSSION THREAD

**Discussion by fastgome** [gopro/labs#778] [General]
I got an engineering machinewithout any markings on the casing. There is no serial number or MAC address in the settings, which makes it impossible for me to connect using GoPro Quik. And the machine's system is extremely unstable. I tried flashing the Labs firmware but all failed. The internal system of the machine shows 10 black, but it appears to be 11 black. I hope someone can provide the correct version to ensure that the machine works properly
![53a2b05f78ce8582d5878e865dd172f](https://github.com/gopro/labs/assets/160385656/ef82bfc0-e65d-4b68-9c09-b734356b7761)
![dbe257c1361a10f87ac94c8b48b4ff9](https://github.com/gopro/labs/assets/160385656/f30dfa2f-3eb0-4259-b5df-ae616d2fde23)

**Comment by dnewman-gpsw** [comment]
That is not a consumer camera, so the published consumer firmware, including Labs, will not run on it.  That type of camera is usually for internal use only.

**Reply by fastgome** [reply]
This looks quite fun. Because it freezes every few minutes during recording,I'm thinking about how I can make it work properly

**Reply by fastgome** [reply]
Would you help me with that please, I have 10 like this and now they can’t even recording anything

**Reply by dnewman-gpsw** [reply]
There is no public firmware for that camera.

**Reply by fastgome** [reply]
Do you know of any other way to solve it？ e.g. using factory firmware

**Reply by dnewman-gpsw** [reply]
I don't.
